<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuracion</class>
 <widget class="QDialog" name="Configuracion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>302</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Configuracion</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../big-data.svg</normaloff>../../big-data.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(141, 255, 139);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Monotype Corsiva</family>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Servidor:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="inDireccion">
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>http://</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Monotype Corsiva</family>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Puerto:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="inPuerto">
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>8080</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);</string>
     </property>
     <property name="title">
      <string>Habilitar cajas</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkCaja1">
        <property name="font">
         <font>
          <family>Lucida Calligraphy</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Caja 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCaja2">
        <property name="font">
         <font>
          <family>Lucida Calligraphy</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Caja 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCaja3">
        <property name="font">
         <font>
          <family>Lucida Calligraphy</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Caja 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCaja4">
        <property name="font">
         <font>
          <family>Lucida Calligraphy</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Caja 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnGuardar">
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ClosedHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(12, 12, 12);</string>
        </property>
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
